app/components/add_reading/index.tsx

```diff
@@ -65,6 +66,44 @@ const AddReading = ({
   }).then(() => {
     cb && cb();
   });
+  const handleCalculateReading = () => {
+    new Promise((resolve, reject) => {
+      let params: {
+        updated_at: string,
+        begin_date?: string,
+        end_date?: string,
+      } = { updated_at: billingItem.updatedAt };
```

Comment:
```suggestion
        updatedAt: string,
        beginDate?: string,
        endDate?: string,
      } = { updatedAt: billingItem.updatedAt };
```
You can use camelcase here, since snakeReq in [/app/readings/api.js](https://github.com/buzzn/admin-app/blob/c0b068d2e1264ceef682b7137ef708d68cc253a9/app/readings/api.js#L30) converts to the correct case.